website/src/segment.tsx

```diff
@@ -411,3 +416,6 @@ const annotationSection = css`
 const storySection = css`
   flex-flow: row wrap;
 `
+const anchor =css`
+  color:black;
+`
```

Comment:
Check the theme.ts file for project wide color and font definitions to use instead of literal values. I'd say if you want a link that doesn't look like one, then it's not the right approach.